src/App.tsx

```diff
@@ -1,26 +1,35 @@
-import React from "react";
-import logo from "./logo.svg";
-import "./App.css";
+import { useEffect, useState } from "react";
+import "./App.scss";
 
-function App() {
-    return (
-        <div className="App">
-            <header className="App-header">
-                <img src={logo} className="App-logo" alt="logo" />
-                <p>
-                    Edit <code>src/App.tsx</code> and save to reload.
-                </p>
-                <a
-                    className="App-link"
-                    href="https://reactjs.org"
-                    target="_blank"
-                    rel="noopener noreferrer"
-                >
-                    Learn React
-                </a>
-            </header>
-        </div>
+import { CssBaseline } from "@mui/material";
+import { OrchestratorForStories } from "./orchestrator/Orchestrator";
+import { ThemeProvider } from "lunatic-edt";
+import Home from "./page/home/Home";
+import "i18n/i18n";
+
+const App = () => {
+    const data = {};
+    const [source, setSource] = useState(null as object | null);
```

Comment:
I'm not very convince by `null as object | null`

Reply:
J'ai essayé de créer le type source pour lunatic mais c'était très complexe. Il faut vraiment une très bonne connaissance du modèle. Laissons object | null pour le moment.